### DISCUSSION TITLE
zh/guide/drivers/baidu_share

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#194] [Announcements]
# zh/guide/drivers/baidu_share

挂载百度网盘分享链接，可供下载，无上传编辑等功能，api非官方。 配置 浏览目录需设置分享surl和pwd，下载还需设置BDUSS。 surl即为分享地址pan.baidu.com/s/后`?前部分，pwd填4位提取码，无提取码留空。 BDUSS由cookie中提取，参考教程 (http://pandownload.net/faq/cookie.htm...

https://alist.nn.ci/zh/guide/drivers/baidu_share.html

**Comment by Daey666** [comment]


**Reply by lixuedong-edu** [reply]
注意这个 surl即为分享地址pan.baidu.com/s/后?前部分
不是全写

**Comment by youtehub** [comment]
我测试过，这个支持浏览器下载的话，最大 48M 的文件，超过就会走客户端下载就是不知道挂载后这个限制是否还存在了

**Comment by 520yxl** [comment]
在线获取百度BDUSS工具 支持5种登录方式：https://tool.cccyun.cc/tool/bduss/index.html
可以自行搭建，源码地址：https://blog.cccyun.cn/post-294.html

**Comment by favoriteacgn** [comment]
为什么我无法删除保存的文件？

**Reply by favoriteacgn** [reply]
是因为文件太大了吗

**Comment by monsoonwind** [comment]
你好，出现这个{"errno":-130,"request_id":8783243700413089867,"errtype":"mis_2","server_time":1692337630,"data":{"title":"","expiredType":0,"link_ctime":0,"fileNums":0,"has_more":false,"list":[]}}

**Comment by outt-iao** [comment]
failed link: failed get link: 403 Forbidden; {"error_code":31360,"error_msg":"expire time out error","request_id":246420242013674469};

**Comment by Sudle2008** [comment]
是不是不能用webdav放这里的视频？

**Comment by bageyalu860** [comment]
failed link: failed get link: 200 OK; {"errno":9019,"errmsg":"needverify","request_id":189206928475359287};浏览器无法正常下载

**Reply by mc-yzy15** [reply]
BDUSS	{"errno":9019,"errmsg":"need verify"}	BDUSS无效

**Comment by tianwen027** [comment]
在下载文件时出现 failed link: failed get link: 200 OK; {"errno":110,"request_id":457695759641759504};
